styles/styles.css

```
@@ -0,0 +1,436 @@
+@font-face {
+  font-family: "PT Sans";
+  font-style: normal;
+  font-weight: 400;
+  src: url(../fonts/ptsans-400.woff2);
+  font-display: swap;
+}
+
+@font-face {
+  font-family: "PT Sans";
+  font-style: normal;
+  font-weight: 700;
+  src: url(../fonts/ptsans-700.woff2);
+  font-display: swap;
+}
+body {
+  font-family: "PT Sans", sans-serif;
+}
+
+.page-header {
+  background: #ffffff;
+}
+
+.bold-style {
+  font-weight: 700;
+}
+
+.navigation-link {
+  font-size: 20px;
+  line-height: 26px;
+  text-align: center;
+  color: #000000;
+  text-decoration: none;
+}
+
+.navigation-item-number-favorites {
+  font-size: 10px;
+  line-height: 13px;
+  color: #FFFFFF;
+}
+
+.navigation-link-current {
+  border-bottom: 2px solid #756257;
+  padding-bottom: 19px;
+}
+
+.navigation-item {
+  list-style-type: none;
+}
+
+.advantages-item {
+  list-style-type: none;
+}
+
+.button-header {
+  background: #756157;
+  border-radius: 4px;
+  color: #FFFFFF;
+  text-transform: uppercase;
+}
+
+.hero {
+  background-color: #E4F1F9;
+  background-image: url(../images/index-background.jpg);
```

```suggestion
  background-image: url("../images/index-background.jpg");
```
https://codeguide.academy/html-css.html#css-quotes

---

исправил